src/main/java/com/google/api/codegen/viewmodel/OptionalArrayMethodView.java

```diff
@@ -57,6 +57,8 @@
 
   public abstract boolean isLongrunningOperation();
 
+  public abstract boolean hasLongrunningReturnValue();
```

Comment:
But LongRunningOperationDetailView is not available in OptionalArrayMethodView. Do you think it is better to construct and add a LongRunningOperationDetailView field, even if only the isEmptyOperation is used?